src/order.cpp

```diff
@@ -45,14 +35,172 @@ namespace py = pybind11;
+ordering should be used. The normal forms for congruence classes are given with
+respect to one of the orders specified by the values in this enum.
+
+The valid values are:
+
+.. py:attribute:: Order.none
+  :value: <Order.none: 0>
+
+  No ordering
+
+.. py:attribute:: Order.shortlex
+  :value: <Order.shortlex: 1>
+
+  The short-lex ordering. Words are first ordered by length, and then
+  lexicographically.
+
+.. py:attribute:: Order.lex
+  :value: <Order.lex: 2>
 
-    py::enum_<Order>(m, "order")
+  The lexicographic ordering. Note that this is not a well-order, so there may
+  not be a lexicographically least word in a given congruence class of words.
+
+.. py:attribute:: Order.recursive
+  :value: <Order.recursive: 3>
+
+  The recursive-path ordering, as described in :cite:`Jantzen2012aa` (Definition
+  1.2.14, page 24).
+)pbdoc")
         .value("none", Order::none)
         .value("shortlex", Order::shortlex)
         .value("lex", Order::lex)
         .value("recursive", Order::recursive);
+
+    m.def(
+        "lexicographical_compare",
+        [](std::string const& x, std::string const& y) {
+          return lexicographical_compare(x, y);
+        },
+        py::arg("x"),
+        py::arg("y"),
+        R"pbdoc(
+:sig=(x: str | List[int], y: str | List[int]) -> bool:
+:only-document-once:
+Compare two values of type :any:`str` or ``List[int]`` using using lexicographical ordering.
+
+:param x: the first object for comparison. 
+:type x: str | List[int]
+
+:param y: the second object for comparison.
+:type y: str | List[int]
+
+:returns: The boolean value ``True`` if *x* is lexicographically less than *y*, and ``False`` otherwise.
+:rtype: bool
+
+:exceptions: This function guarantees not to throw a ``LibsemigroupsError``.
+
+:complexity: At most :math:`O(n)` where :math:`n` is the minimum of the length of *x* and the length of *y*.
+)pbdoc");
+
+    m.def(
+        "lexicographical_compare",
+        [](word_type const& x, word_type const& y) {
+          return lexicographical_compare(x, y);
+        },
+        py::arg("x"),
+        py::arg("y"),
+        R"pbdoc(
+:sig=(x: str | List[int], y: str | List[int]) -> bool:
+:only-document-once:
+)pbdoc");
+
+    m.def(
+        "shortlex_compare",
+        [](std::string const& x, std::string const& y) {
+          return shortlex_compare(x, y);
+        },
+        py::arg("x"),
+        py::arg("y"),
+        R"pbdoc(
+:sig=(x: str | List[int], y: str | List[int]) -> bool:
+:only-document-once:
+Compare two values of type :any:`str` or ``List[int]`` using shortlex ordering.
+
+:param x: the first object for comparison. 
+:type x: str | List[int]
+
+:param y: the second object for comparison.
+:type y: str | List[int]
+
+:returns: The boolean value ``True`` if *x`* is short-lex less than *y*, and ``False`` otherwise.
+:rtype: bool
+
+:exceptions: This function guarantees not to throw a ``LibsemigroupsError``.
+
```

Comment:
I think we weren't including these lines in the python doc

Reply:
Ah my bad. I'll change it in this PR and open an issue to get rid of it elsewhere (this isn't the first time I've included them).